src/core/ddsi/tests/sysdeps.c

```diff
@@ -199,7 +199,10 @@ CU_Test (ddsi_sysdeps, log_stacktrace_other, .init = setup, .fini = teardown)
   CU_ASSERT_FATAL (rc == 0);
 
   ddsrt_mutex_lock (&arg.lock);
+  /* coverity[loop_top: FALSE] */
+  /* coverity[exit_condition: FALSE] */
   while (!arg.incallback)
+    /* coverity[loop_bottom: FALSE] */
```

Comment:
I would prefer adding { } here. I love my dangling statements, but not when they aren't immediately below the conditional.

Reply:
No problem